websites/P/Puray Animes/presence.ts

```diff
@@ -0,0 +1,53 @@
+const presence = new Presence({
+		clientId: "972246349917610054"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", () => {
+	const { pathname } = window.location,
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		};
+	if (document.querySelector("input[id^=headlessui]"))
+		presenceData.details = "Pesquisando Animes";
+	else if (pathname.startsWith("/home"))
+		presenceData.details = "Na página principal";
+	else if (pathname.startsWith("/profile/")) {
+		presenceData.details = "Visualizando perfil:";
+		presenceData.state = `${
+			document.querySelector("h3").childNodes[0]?.textContent
+		} - ${document.querySelector("button[class*=blue] span")?.textContent}`;
+		presenceData.buttons = [{ label: "Ver perfil", url: location.href }];
+	} else if (pathname.startsWith("/anime/")) {
+		const genres: string[] = [];
+		for (const item of document.querySelectorAll(
+			"div[class^=sm]>div[class^=mb]"
+		))
+			genres.push(item.textContent);
+		presenceData.details = document.querySelector(
+			"section div[class^=text-3xl]"
+		)?.textContent;
+		if (genres.length) presenceData.state = genres.join(", ");
```

Comment:
I personally don't think showing the genres in the state is relevant information, maybe you could move the anime title here and have a generic phrase in the details like "Watching anime:"

Reply:
Personally I found it better this way, but in the future I can change to a more generic phrase too, no problem.

Reply:
What do you mean in the future?

Reply:
I mean that I can still change the presence according to the evolution of the site, adding more features.

Reply:
> What do you mean in the future?

Anyway, I fixed what you asked for and removed the genres.